partner_statement/report/activity_statement.py

```diff
@@ -140,9 +141,9 @@ def _get_report_values(self, docids, data):
         if not data:
             data = {}
         if 'company_id' not in data:
-            wiz = self.env["activity.statement.wizard"].with_context(
+            wiz = Form(self.env["activity.statement.wizard"].with_context(
```

Comment:
Don't! This `Form` server was created only for testing. That's why it is found in odoo.tests 😏

Reply:
@mreficent it's on tests, but it's used in other parts. I see this better than simply call all the onchanges you know (and leaving unknown onchanges), or to depend on `onchange_helper` module, that is buggy in some cases (when there's a context in fields).

Reply:
@mreficent In this particular case, it is the precise behaviour desired. As though the user opened the form, the defaults were populated, and triggered the onchanges. I figured if it is considered good enough for core for basically the same cases, then it must be OK.